### PR TITLE
abranch

### PR DESCRIPTION
**Pull request by veronikatimkina**

### CODE REVIEW

File: src/TestBr.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class TestBr {
     public static void main(String[] args) {
-
+        System.out.println();
```
Comment (nika-qa) [inline src/TestBr.java:3]: comment from web

Comment (veronikatimkina) [inline src/TestBr.java:3]: from ide

**Review by nika-qa — COMMENTED** [review]
review comment

File: src/TestBr.java (hunk 1 above)
Comment (nika-qa) [reply src/TestBr.java:3]: reply from web

Comment (nika-qa) [inline src/TestBr.java:3]: from IDE

File: src/main/java/pack/Test.java [hunk 2]
```diff
@@ -1,7 +1,5 @@
 package main.java.pack;
```
Comment (veronikatimkina) [inline src/main/java/pack/Test.java:1]: from web

**Review by nika-qa — CHANGES_REQUESTED** [review]

File: src/main/java/pack/Test.java [hunk 3]
```diff
@@ -1,7 +1,5 @@
...
 
-import org.w3c.dom.ls.LSOutput;
-
 public class Test {
```
Comment (veronikatimkina) [inline src/main/java/pack/Test.java:5]: start

**Pull request closed without merging**